git_machete/github.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+import http.client
+import json
+import os
+import re
+from http.client import HTTPResponse, HTTPSConnection
+from typing import Dict, List, Optional, Any, Tuple
+
+from git_machete.cmd import MacheteException, fmt
+
+
+class GitHubPullRequest(object):
+    def __init__(self, number: int, user: str, base: str, head: str):
```

Comment:
Does it pass `mypy`? :thinking: Or maybe `mypy` is not execute for this file? 

Cause IIRC the __init__ shall be typed as ` -> None`...

Reply:
It is executed, definitely... apparently it doesn't complain about c'tor not being typed? did it complain to you?

Reply:
hmmm... :thinking: possibly I mistyped the constructor with `'ClassName'` instead of `'None'` and that's from my error came